en/docs/setup/changing-the-hostname.md

```diff
@@ -122,7 +100,7 @@ you can use the same public and private key pair to generate a new CSR
 with the updated CN (subject). This can be done by adding the "-dname"
 option when "-certreq" is executed. Once the CA certificate is obtained,
 follow the instructions given in the [importing certificates to the
-keystore](https://docs.wso2.com/display/ADMIN44x/Creating+New+Keystores#CreatingNewKeystores-Step2:Importingcertificatestothekeystore)
+keystore](../../admin-guide/creating-new-keystores#step-2:-importing-certificates-to-the-keystore) 
 topic to import it correctly. By doing that, you do not need to touch
```

Comment:
this anchor link will not work without an anchor tag added to the creating-new-keystores .md file.

Reply:
As the referred section is a subheading, it will automatically act as an anchor tag.